packages/framework/src/Form/Admin/Product/Availability/AvailabilityFormType.php

```diff
@@ -35,6 +36,8 @@ public function buildForm(FormBuilderInterface $builder, array $options)
                 'required' => false,
                 'invalid_message' => 'Number of days to expedite must be whole number.',
                 'constraints' => [
+                    new Constraints\NotBlank(['message' => 'Please enter number of days to dispatch. For out of stock fill 65535 as maximal value.']),
```

Comment:
This is really weird to have out of stock availability set to 179,55 years. Why not keep null? You can work with null in order by https://www.postgresql.org/docs/current/queries-order.html

Reply:
in the framework we have an unbeatable generic setting in SortableNullsWalker. After agreement with PK, I chose this solution.

Reply:
found more pretty solution